usecases/evaluate_scenario/preprocessing.go

```diff
@@ -22,7 +22,10 @@ type ScreeningPreprocessor func(
 	scc models.SanctionCheckConfig,
 ) ([]models.OpenSanctionsCheckQuery, error)
 
-func SkipIfUnder(ctx context.Context, e ScenarioEvaluator, queries []models.OpenSanctionsCheckQuery, iteration models.ScenarioIteration, scc models.SanctionCheckConfig) ([]models.OpenSanctionsCheckQuery, error) {
+func SkipIfUnder(ctx context.Context, e ScenarioEvaluator, queries []models.OpenSanctionsCheckQuery,
+	iteration models.ScenarioIteration, scc models.SanctionCheckConfig,
+) ([]models.OpenSanctionsCheckQuery, error) {
+	// nitpicking, but it's not "really" an edge case
```

Comment:
Not sure I understand the comment. This is to skip when in the "disabled" state (which is `0`, since it doesn't really make sense to skip things with 0 length).